pkgs/nodeinfo/nodeinfo.sh

```diff
@@ -2,14 +2,21 @@ set -e
 set -o pipefail
 
 BITCOIND_ONION="$(cat /var/lib/onion-chef/operator/bitcoind)"
-CLIGHTNING_NODEID=$(sudo -u clightning lightning-cli --lightning-dir=/var/lib/clightning getinfo | jq -r '.id')
-CLIGHTNING_ONION="$(cat /var/lib/onion-chef/operator/clightning)"
-CLIGHTNING_ID="$CLIGHTNING_NODEID@$CLIGHTNING_ONION:9735"
-
 echo BITCOIND_ONION="$BITCOIND_ONION"
-echo CLIGHTNING_NODEID="$CLIGHTNING_NODEID"
-echo CLIGHTNING_ONION="$CLIGHTNING_ONION"
-echo CLIGHTNING_ID="$CLIGHTNING_ID"
+
+if [ -x "$(command -v clightning)" ]; then
+    CLIGHTNING_NODEID=$(sudo -u clightning lightning-cli --lightning-dir=/var/lib/clightning getinfo | jq -r '.id')
+    CLIGHTNING_ONION="$(cat /var/lib/onion-chef/operator/clightning)"
+    CLIGHTNING_ID="$CLIGHTNING_NODEID@$CLIGHTNING_ONION:9735"
+    echo CLIGHTNING_NODEID="$CLIGHTNING_NODEID"
+    echo CLIGHTNING_ONION="$CLIGHTNING_ONION"
+    echo CLIGHTNING_ID="$CLIGHTNING_ID"
+fi
+
+if [ -x "$(command -v lncli)" ]; then
+    LND_NODEID=$(sudo -u lnd lncli --tlscertpath /secrets/lnd_cert --macaroonpath /var/lib/lnd/chain/bitcoin/mainnet/admin.macaroon getinfo | jq -r '.uris[0]')
+    echo LND_NODEID="$LND_NODEID"
```

Comment:
The onion address reported by `lncli getinfo` is different by that found in `/var/lib/onion-chef/operator/lnd`. I didn't dig into it yet, I don't know how lnd uses `ControlPort` to create services, or what `tor.streamisolation=true` implies. For now I just included the node id reported by `lncli`.

Reply:
Good catch. It seems like in that case we could remove the explicit hidden service configuration and remove lnd from the onion chef to avoid confusion.